File: median_of_two_sorted_arrays/median_of_two_sorted_arrays.rb
def find_median_sorted_arrays(nums1, nums2)
    answer = (nums1+nums2).sort
    if answer.length.odd?
        answer[answer.length/2]
    else
        idx = answer.length/2
        (answer[idx-1] + answer[idx])/2.0
    end
end

File: median_of_two_sorted_arrays/medianOfTwoSortedArrays.js
const findMedianSortedArrays = (nums1, nums2) => {
	answer = nums1.concat(nums2).sort((a,b)=> {
		return a-b
	});
	if (answer.length % 2 !== 0) {
		idx = Math.floor(answer.length/2)
		return answer[idx];
	} 
	idx = answer.length/2
	return (answer[idx-1] + answer[idx])/2.0;
};
